pandas/tests/frame/test_query_eval.py

```diff
@@ -202,6 +202,13 @@ def test_eval_simple(self, engine, parser):
         expected = df["a"]
         tm.assert_series_equal(expected, res)
 
+    def test_extension_array_eval():
```

Comment:
Where this is indented, it is part of a class. It needs `self` as a first argument.